src/_assets/css/_includes/bootstrap-overrides.scss

```diff
@@ -5,7 +5,7 @@ $small-font-size: 85%;
 $h1-font-size: 2em;
 $h2-font-size: 1.5em;
 $h3-font-size: 1.125em;
-$h4-font-size: 1.075em;
+$h4-font-size: 1em;
```

Comment:
This is good, but I can't fail but notice that now `h4`, `h5` and `h6` are all the same size :)

And we do have cases in the docs that we use as many as 5 nesting levels :/

Might be the time to loop in a designer and review the header styles. (Or stop nesting so many levels in a single page, which is also humanly hard to follow/distinguish)

Reply:
That is a good point, Rodolfo, thank you. My preference is that we do not ever use an H5 or H6. Doing so is, as you note, not super helpful to readers because they're those headers are visually so close. In reality, the need for this level heading suggests there's actually a deeper content problem - we shouldn't need more than four headings.